src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -64,13 +74,47 @@ public override bool Equals(System.Object obj)
               eventTime == sec.eventTime;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator ==(GestureInfoType arg1, GestureInfoType arg2)
```

Comment:
In NUI, it is normal to check nullity of each parameters.
Please refers `Rectangle` class.